lldb/test/API/lang/swift/variables/optionals/TestSwiftOptionals.py

```diff
@@ -99,6 +103,27 @@ def do_check_api(self):
             num_children=1)
         uoptString_Some.GetChildAtIndex(99)
 
+        optTrue = self.frame().FindVariable("optTrue")
+        lldbutil.check_variable(
+            self,
+            optTrue,
+            use_dynamic=False,
+            num_children=1)
```

Comment:
Wouldn't it be good to specify a `summary` to check?

Reply:
Do you have an example of that? Do you mean jut `v`?

Reply:
so I think the tests would have `summary="{true,false,nil}"`

Reply:
TIL. Thanks!